docs/.nav.yml

```diff
@@ -4,7 +4,11 @@
 nav:
   - overview
   - Getting Started: getting_started
-  - concepts
-  - guides
-  - API Documentation: reference
-  - Release Notes: release_notes.md
+  - "Documentation: Concepts": concepts
+  - "Documentation: API Reference": reference
+  - "Documentation: Guides": guides
+  - "Documentation: Upgrading": upgrading
```

Comment:
Here you can see how to "2-level" tabs work - The section / tab name are actually like `"Documentation: Concepts"`.

But when rendering the top navigation tabs, the trick I did was to split and group the tab names by prefixes. So all tabs that start with  `Documentation: ` will be stashed away under the `Documentation` top-level tab.

But in the remained of the app we still display these pages as they truly are, like `"Documentation: Concepts"`. You can see that in the sidebar navigation, which have those full names:

<img width="447" height="803" alt="Screenshot 2025-08-16 at 16 01 42" src="https://github.com/user-attachments/assets/2a8977fb-f80d-4268-a969-866de7fde877" />